.eslintrc.json

```diff
@@ -1,7 +1,8 @@
 {
   "parser": "babel-eslint",
   "env": {
-    "browser": true
+    "browser": true,
+    "jest": true
```

Comment:
Should we pull this into overrides so it only affects `*.test.*` files?

Reply:
Probably - I think a lot of the jest-specific syntax only happens in `*.test.js` files, but we can be double safe 👍